### ISSUE TITLE
Unitree Z1 Arm Fuse Blows

### ISSUE THREAD

**Issue by AleksandrKashirin** [unitreerobotics/z1_sdk#7]
Dear Unitree,

During my work with Unitree Z1 manipulator, I have faced such a problem that switching from high level command mode to low level command mode leads to the unpredictable jerky movement behavior of the robot, which further leads to blowing the fuse on the motherboard. Could you please share your experience and justify the reasons why this happens and what could be done to prevent such behavior?

With kind regards, 
Aleksandr

**Comment by AleksandrKashirin** [comment]
Hi, @foxp1!
I have replaced the fuse with help of supplied company.
After that I decided to use laboratory power supply, where you can manually reduce the output current down to the nominal of the fuse (10A or less).
![image](https://github.com/unitreerobotics/z1_sdk/assets/40818765/1809d989-3782-4a95-ab95-7489155e7b8e)

**Comment by surajhanchinal** [comment]
+1 faced a similar issue today, when switching between modes

**Comment by surajhanchinal** [comment]
@foxp1 Were you able to resolve this issue?

**Comment by foxp1** [comment]
@AleksandrKashirin Thanks for the tip. It was indeed the same issue for us. I am waiting to hear from Unitree on what the next steps should be as the arm is still under warranty.

@surajhanchinal I'll let you know as soon as I have more information. Have you resolved it already on your end?

**Comment by surajhanchinal** [comment]
@foxp1 , Yes we did manage to resolve the issue. Unitree provided a replacement board for us but you should also be able to replace the fuse inside and have it work. There are 2 SMD fuses with a rating of 125V and 10A.

**Comment by bk-poudel** [comment]
I think we are also in a similar situation right now. How to solve that issue? Or how to contact them.
